### PR TITLE
access the container inside the customized Standard Controller

### PR DESCRIPTION
**Pull request by davidroberto**
| Q               | A
| --------------- | -----
| Branch?         | 1.2, 1.3 or master 
| Bug fix?        | no/yes
| New feature?    | no/yes
| BC breaks?      | no/yes
| Deprecations?   | no/yes 
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

File: docs/customization/controller.rst [hunk 1]
```diff
@@ -188,4 +188,8 @@ Remember to import the ``app/config/services.yml`` into the ``app/config/config.
 
 From now on your ``customAction`` of the ``HomepageController`` will be available alongside the ``indexAction`` from the base class.
 
+.. use services::
+
+If you need to access the container inside your action, your customized Standard Controller needs to extends ``Symfony\Bundle\FrameworkBundle\Controller\Controller``
```
Comment (lchrusciel) [inline docs/customization/controller.rst]: This is a bad practice. We should recommend using DI instead.

Comment (davidroberto) [reply docs/customization/controller.rst]: I agree about the use of the container, but Sylius makes heavy use of it in it's doc for all the services:
https://docs.sylius.com/en/1.2/book/architecture/resource_layer.html

Comment (Zales0123) [reply docs/customization/controller.rst]: I think that these docs are written this way just to make things shorter and easier to read. But they can be changed too, to not spread the wrong practices :)

Comment (pamil) [reply docs/customization/controller.rst]: Anyway, you don't need to extend Symfony's Controller to get the container, you could just inject it instead.

**Pull request closed without merging**